bundle/src/main/java/com/adobe/acs/commons/models/injectors/impl/ContentPolicyValueInjector.java

```diff
@@ -61,15 +77,25 @@ public String getName() {
     @Nullable
     @Override
     public Object getValue(Object adaptable, String name, Type declaredType, AnnotatedElement element,
-                           DisposalCallbackRegistry callbackRegistry) {
-        ContentPolicy policy = getContentPolicy(adaptable);
+            DisposalCallbackRegistry callbackRegistry) {
+        if (!config.enabled()) {
+            return null;
+        }
 
+        ContentPolicy policy = getContentPolicy(adaptable);
         if(policy != null){
             return convertValueMapValue(policy.getProperties(), name, declaredType);
         }
-
         return null;
     }
 
 
+    @ObjectClassDefinition(name="ACS AEM Commons ContentPolicyValueInjector")
+    public @interface Configuration {
+
+        @AttributeDefinition(name="enabled")
+        public boolean enabled() default true;
```

Comment:
Even with the risk of breaking existing consumers I would switch to false (i.e. opt in) like for all other ACS AEM commons features and just mention this in the release notes

Reply:
@kwin +1

@niekraaijmakers as you have added this injector a year ago, do you have any insights into its usage?